support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -339,7 +339,8 @@ export function ThreeTierLanding({
 	 */
 	const campaignSettings = getCampaignSettings(countryGroupId);
 	const enableSingleContributionsTab =
-		campaignSettings?.enableSingleContributions;
```

Comment:
This change means that for now it's not configurable with the campaign. This keeps things simple. We can change it back later.